game.js

```diff
@@ -0,0 +1,13 @@
+let trumps = document.querySelectorAll('.trump');
+let score = 0;
+console.log(score);
```

Comment:
Remove any calls to the console. This is code meant for development only, not for production!